app/javascript/components/admin/AdminUserRow.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Stack } from 'react-bootstrap';
+import Avatar from '../users/Avatar';
+
+export default function AdminUserRow({ user }) {
+  return (
+    <tr key={user.id} className="align-middle text-muted">
+      <td className="text-dark">
+        <Stack direction="horizontal">
+          <div>
+            <Avatar className="avatar" avatar={user.avatar} radius={40} />
+          </div>
```

Comment:
You can add <div className="me-2"> to avatar parent here instead of having an extra CSS class.